lib/browser/jasmine-patch.js

```diff
@@ -0,0 +1,3 @@
+var jasminePatch = require('../jasmine/patch');
+
+jasminePatch.apply();
```

Comment:
The idea of this 'browser' folder is to generate IIFE in the `dist/` folder.

All other CJS module module in the `lib/` folder are not immediately invoked, ie you can require them and invoke them when you want in a CJS context.

Does this make sense or do you think that importing the module should also apply the patch ?